### ISSUE TITLE
Add prompting for troubleshoot command for better results when in screen reader mode

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode-copilot-vision#69]
The model should be told when in screen reader mode so it provides keyboard friendly guidance etc.
